plugins.star

```diff
@@ -14,11 +15,14 @@ plugins_map = {
     "AutoGPTRandomValues": {"repository": STDLIB_PLUGIN_REPO, "branch": MASTER_BRANCH},
     "AutoGPTSpacePlugin": {"repository": STDLIB_PLUGIN_REPO, "branch": MASTER_BRANCH},
     # end of standard plugins
-    "AutoGPTGoogleAnalyticsPlugin": {"repository": "isaiahbjork/Auto-GPT-Google-Analytics-Plugin", "branch": MASTER_BRANCH},
     "AutoGPTAlpacaTraderPlugin": {"repository": "danikhan632/Auto-GPT-AlpacaTrader-Plugin", "branch": MASTER_BRANCH},
     "AutoGPTUserInput": {"repository": "HFrovinJensen/Auto-GPT-User-Input-Plugin", "branch": MASTER_BRANCH},
-    "BingAI": {"repository": "gravelBridge/AutoGPT-BingAI", "branch": MASTER_BRANCH},
+    "AutoGPTBingAI": {"repository": "gravelBridge/AutoGPT-BingAI", "branch": MAIN_BRANCH},
     "AutoGPTCryptoPlugin": {"repository": "isaiahbjork/Auto-GPT-Crypto-Plugin", "branch": MASTER_BRANCH},
+    "AutoGPTDiscordPlugin": {"repository": "gravelBridge/AutoGPT-Discord", "branch": MAIN_BRANCH},
+    "AutoGPTDollyPlugin": {"repository": "pr-0f3t/Auto-GPT-Dolly-Plugin", "branch": MASTER_BRANCH},
+    "AutoGPTGoogleAnalyticsPlugin": {"repository": "isaiahbjork/Auto-GPT-Google-Analytics-Plugin", "branch": MASTER_BRANCH},
+    "AutoGPTIFTTTWebhooksPlugin": {"repository": "AntonioCiolino/AutoGPT-IFTTT", "branch": MASTER_BRANCH},
```

Comment:
this one needs to be AutoGPT_IFTTT based on https://github.com/AntonioCiolino/AutoGPT-IFTTT/blob/master/src/AutoGPT_IFTTT/__init__.py#L19

Reply:
I made sure the install worked but did not verify with the app itself.  Will make the changes.

Reply:
I validated each plugin by starting autogpt and making sure I can get to the prompt.